jujugui/static/gui/src/app/components/deployment-flow/payment/payment.js

```diff
@@ -88,12 +288,35 @@ YUI.add('deployment-payment', function() {
     },
 
     /**
+      Format the credit card number.
+
+      @method _formatCardNumber
+    */
+    _formatCardNumber: function() {
+      // Get the current number and clean all the spaces.
+      const number = this.refs.cardNumber.getValue().replace(/ /g, '');
+      let parts = [];
+      // Loop through the blocks of four numbers.
```

Comment:
I propose removing this entirely or in favour of adding a more sophisticated implementation/lib as not all CC's use 4 digit chunking.